src/cheri/insns/ysunseal_32bit.adoc

```diff
@@ -0,0 +1,74 @@
+<<<
+
+[#YSUNSEAL,reftext="{YSUNSEAL}"]
+==== {YSUNSEAL}
+
+#Begin new since last ARC review#
+
+include::new_encoding_note.adoc[]
+
+Synopsis::
+{YSUNSEAL_DESC}
+
+Mnemonic::
+`{ysunseal_lc} {cd}, {cs1}, {cs2}`
+
+Encoding::
+include::wavedrom/ysunseal.adoc[]
+
+Description::
+Copy `{cs2}` to `{cd}`.
++
+Set `{cd}.ct=0`.  (That is, unseal `{cd}`.)
++
+Set `{cd}.tag=1` if:
++
+. `{cs1}.tag` is set, and
+. `{cs1}` passes all <<section_cap_integrity,integrity>> checks, and
+. `{cs1}` is not sealed (that is, `{cs1}` has zero <<sec_cap_type>>), and
+. `{cs2}.tag` is set, and
+. `{cs2}` passes all <<section_cap_integrity,integrity>> checks, and
+. `{cs2}` is sealed (that is, `{cs2}` has non-zero <<sec_cap_type>>), and
+. `{cs2}` 's permissions and bounds are equal to or a subset of `{cs1}` 's, and
+. any extension-specific constraints on <<YSUNSEAL>> hold.
+
++
+Otherwise, set `{cd}.tag=0`
+
+NOTE: When `{cs1}` is `{creg}0` <<YSUNSEAL>> will copy `{cs2}` to `{cd}` and clear `{cd}.tag` and `{cd}.ct`.
+However future extensions may add additional behavior to update currently reserved fields,
+and so software should not assume `{cs1}==0` to be a pseudo-instruction for {ctag} and type clearing.
+
+NOTE: <<YSUNSEAL>> is essentially <<CBLD>> except that it
+requires `{cs2}` to be tagged and always sets `{cd}.ct` to `0`.
+
+[NOTE]
+=====
+<<YSUNSEAL>> is intended to enable "superset unsealing":
+wherein software, given a capability in `{cs2}`, can present, in `{cs1}`,
+a capability granting a superset of the authority
+that would be granted by an unsealed version of the capability in `{cs2}`,
+and thereby reconstruct, into `{cd}`, that unsealed version of `{cs2}`.
+For example, to unseal from `x3` into `x1`
```

Comment:
Don't we need `{creg}` instead of `x` throughout the rest of this?

Reply:
Deleted instead.